src/components/PlayClassCard/PlayClassCard.js

```diff
@@ -207,27 +215,38 @@ export default function PlayClassCard({ classToPlay, toggleFavourite }) {
           }
         />
       </div>
-      <h3>Preparation Time</h3>
-      <div className="PrepTime">
-        <div className="PrepTime-line"></div>
-        <div className="PrepTime-line__time">
-          <p className="PrepTime-line__time--text">0</p>
-          <p className="PrepTime-line__time--text">
-            {prepTime} sec ({prepFormattedTime})
-          </p>
-        </div>
-      </div>
-      <h3>Class</h3>
+      <h3>Preparation Time: {prepTime} sec</h3>
+
+      <p className="PrepTime-line__time--text">{prepFormattedTime}</p>
+      <div className="PrepTime-line"></div>
+      <div className="PrepTime-line__time"></div>
+
+      <h3>
+        Class:{" "}
```

Comment:
The prettier extension should stop doing that :D

Reply:
Thanks for catching that. I actually have the prettier installed. Will take a look.